src/applications/search/containers/SearchApp.jsx

```diff
@@ -16,6 +18,10 @@ import IconSearch from '@department-of-veterans-affairs/formation-react/IconSear
 import Pagination from '@department-of-veterans-affairs/formation-react/Pagination';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
+import SearchBreadcrumbs from '../components/SearchBreadcrumbs';
+
+const SCREENREADER_FOCUS_CLASSNAME = 'sr-focus';
```

Comment:
This class is applied to multiple elements. The helper `focusElement` will apply browser focus to the first one.